demo/custom_path/run.py

```diff
@@ -5,13 +5,18 @@
 
 app = FastAPI()
 
+
 @app.get("/")
 def read_main():
     return {"message": "This is your main app"}
 
+
 io = gr.Interface(lambda x: "Hello, " + x + "!", "textbox", "textbox")
 gradio_app = gr.routes.App.create_app(io)
+app = gr.mount_gradio_app(app, gradio_app, server_name="localhost", port="8000", path="/gradio")
```

Comment:
A suggestion for this function: can we have it take a Gradio Blocks object instead of a FastAPI app as its second parameter? And then internally it calls it `gr.routes.App.create_app(io)`. This saves our users a line of code

Reply:
Sure thing! Good point.